Source/DafnyCore/AST/MemberDecls.cs

```diff
@@ -15,7 +15,10 @@ public abstract class MemberDecl : Declaration {
     }
   }
 
-  public readonly bool IsOpaque;
+  public virtual bool IsOpaque {
+    get { return false; }
+  }
```

Comment:
C# syntax allows you to write this as

```suggestion
  public virtual bool IsOpaque => false;
```